orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/ExecutionRunner.java

```diff
@@ -21,22 +21,21 @@
 import javax.annotation.Nullable;
 
 public interface ExecutionRunner {
-  void start(@Nonnull Execution execution) throws Exception;
+  void start(@Nonnull Execution execution);
 
-  default void restart(@Nonnull Execution execution, @Nonnull String stageId) throws Exception {
+  default void restart(@Nonnull Execution execution, @Nonnull String stageId) {
     throw new UnsupportedOperationException();
   }
 
-  default void reschedule(@Nonnull Execution execution) throws Exception {
+  default void reschedule(@Nonnull Execution execution) {
```

Comment:
shouldn't these be `throws UnsupportedOperationException`? I guess it doesn't matter because it's an interface?

Reply:
yeah and also UnsupportedOperationException is an unchecked exception